.pipelines/e2e.yml

```diff
@@ -55,5 +55,6 @@ jobs:
       register_sub
 
       export CI=true
+      export E2E_DELETE_CLUSTER=false
```

Comment:
wondering if we should document the use of this env var outside of it just existing within `test/e2e/setup.go` and having to determine where to plumb it in every time :thinking: